modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -106,9 +106,10 @@ public class HeadlessBuilderResourceTest extends BaseTestCase {
 
 	@BeforeClass
 	public static void setUpClass() {
-		_dateFormat = DateFormatFactoryUtil.getSimpleDateFormat("yyyy-MM-dd");
+		_dateFormat = DateFormatFactoryUtil.getSimpleDateFormat(
+			"yyyy-MM-dd'T'00:00:00'Z'");
 		_dateTimeFormat = DateFormatFactoryUtil.getSimpleDateFormat(
-			"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+			"yyyy-MM-dd'T'HH:mm:ss.SSS");
```

Comment:
If I use the Object's REST API, the date time format is different:

```
curl -X GET -u test@liferay.com:test http://localhost:8080/o/c/tests?fields=dateField,dateTimeField
```
```
{
  "actions" : {
    "updateBatch" : {
      "method" : "PUT",
      "href" : "http://localhost:8080/o/c/tests/batch"
    },
    "create" : {
      "method" : "POST",
      "href" : "http://localhost:8080/o/c/tests/"
    },
    "createBatch" : {
      "method" : "POST",
      "href" : "http://localhost:8080/o/c/tests/batch"
    },
    "deleteBatch" : {
      "method" : "DELETE",
      "href" : "http://localhost:8080/o/c/tests/batch"
    }
  },
  "facets" : [ ],
  "items" : [ {
    "dateField" : "2023-11-26T00:00:00Z",
    "dateTimeField" : "2023-11-28T17:29:00.000Z"
  } ],
  "lastPage" : 1,
  "page" : 1,
  "pageSize" : 20,
  "totalCount" : 1
}
```

Which one should we use?